tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -169,8 +171,8 @@ def test_export_saturatuion_fix(disabled):
     activation_quantizers_be, weight_quantizers_be = get_quantizers(compression_model)
     ref_weight_qspec = TFQuantizerSpec(mode=QuantizationMode.SYMMETRIC,
                                        num_bits=8,
-                                       signedness_to_force=None,
-                                       per_channel=False,
+                                       signedness_to_force=True,
+                                       per_channel=True,
```

Comment:
Could you provide a short explanation for the changes?

Reply:
The default target device maps to CPU, which impacts the Conv2D's resulting weight quantization selected by default, which is signed per-channel.